.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next/core-web-vitals", "prettier"]
```

Comment:
This overrides specific lint rules that can conflict with prettier (otherwise you end up with prettier trying to format code one way, and then eslint complaining and trying to format it another way). 

With this, prettier is in charge of all formatting.

Reply:
Good!